{"name": "Принять заказ. Если передан id заказа, который уже взят в работу, НЕ успешно", "status": "passed", "start": 1712509320939, "stop": 1712509321566, "uuid": "099512a0-c6cd-4d0e-933d-b4a06ce93ab4", "historyId": "7f97b29033a10ddc075e4866a108522e", "testCaseId": "7f97b29033a10ddc075e4866a108522e", "fullName": "tests.orders.test_accept_order.TestAcceptOrder#test_accept_order_with_accepted_order_id_fail", "labels": [{"name": "parentSuite", "value": "tests.orders"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "DESKTOP-HCED1QB"}, {"name": "thread", "value": "35896-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.orders.test_accept_order"}]}